app/(components)/navigation.jsx

```
@@ -14,15 +15,15 @@ export default function Navigation() {
+            <button type="button" className={styles.faBars} onClick={handleClick}>
                 <FontAwesomeIcon icon={faBars}/>
             </button>
-            <ul className={"menu"} data-opened={isOpened}>
-                <button type="button" className="faX" onClick={handleClick}>
+            <ul className={styles.menu} data-opened={isOpened}>
+                <button type="button" className={styles.faX} onClick={handleClick}>
                     <FontAwesomeIcon icon={faX}/>
                 </button>
-                <li onClick={handleClick}><Link href="/">Home</Link></li>
-                <li onClick={handleClick}><Link href="/">My Work</Link></li>
+                <li onClick={handleClick}><Link href="/public">Home</Link></li>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)